src/plugins/data_views/common/fields/fields.mocks.ts

```diff
@@ -284,6 +284,17 @@ export const fields: FieldSpec[] = [
     aggregatable: true,
     readFromDocValues: false,
   },
+  {
```

Comment:
Why is this change needed?

Reply:
I can take this out actually. It was needed before I updated the `validatePotentialWildcardInput` function because I was only returning a warning when validating the wildcard input if the field was `file.path.text`. But now that the function will return a warning regardless of the field (warning will tell the users that wildcard usage may cause performance issues), the tests should work as expected